runtime/bcutil/defineclass.c

```diff
@@ -761,20 +761,19 @@ callDynamicLoader(J9VMThread *vmThread, J9LoadROMClassData *loadData, U_8 * inte
 			FALSE, /* isIntermediateROMClass */
 			localBuffer);
 
-	/* The module of a class transformed by a JVMTI agent needs access to unnamed modules */
-	if ((J2SE_VERSION(vm) >= J2SE_V11)
-		&& (classFileBytesReplacedByRIA || classFileBytesReplacedByRCA)
-		&& (NULL != loadData->romClass)
-	) {
-		J9Module *module = J9_VM_FUNCTION(vmThread, findModuleForPackage)(vmThread, loadData->classLoader,
-				loadData->className, (U_32) packageNameLength(loadData->romClass));
-		if (NULL != module) {
-			module->isLoose = TRUE;
+	if (BCT_ERR_NO_ERROR == result) {
+		/* The module of a class transformed by a JVMTI agent needs access to unnamed modules */
+		if ((J2SE_VERSION(vm) >= J2SE_V11)
+				&& (classFileBytesReplacedByRIA || classFileBytesReplacedByRCA)
```

Comment:
Minor nitpick - this line of the if statement is indented to far.  It should be a single tab (or preferably in my view none!).
```suggestion
		&& (classFileBytesReplacedByRIA || classFileBytesReplacedByRCA)
```

Reply:
Seems Eclipses likes to indent by 2 tabs. I've removed one.